docs/DESIGN.md

```diff
@@ -14,7 +14,7 @@ Linear base consists of the following:
 * tools ported from [`base`] and from other critical haskell
   libraries, like `lens`,
 * new APIs for using system resources, e.g., file I/O in
-  [`System.IO.Resource`],
+  [`System.IO.Resource.Linear`],
```

Comment:
This one is a breaking change. We will have to communicate on it.

Reply:
We need to write it down in the changelog, I suppose? Do we need to do something else?

(I am referencing the changelog issue there to not forget: #377 )

Reply:
Yes, in the change log. All the breaking change will have to be clearly identified.